README.md

```diff
@@ -64,17 +64,47 @@ There are two ways you can use Kedro-Viz:
 
 ## Usage
 
-#### Compatibility with Kedro
+#### Compatibility with Kedro and Kedro-datasets   
+
+Ensure your `Kedro`, `Kedro-Viz` and `Kedro-datasets` versions are supported by referencing the following table:
+
+<table>
+    <tr>
+        <th>Python Version</th>
+        <th style="text-align: center" colspan="3">Last Supported</th>
+    </tr>
+    <tr>
+        <td></td>
+        <td>Kedro</td>
+        <td>Kedro-Viz</td>
+        <td>Kedro-datasets</td>
+    </tr>
+    <tr>
+        <td>3.6</td>
+        <td>0.17.7</td>
+        <td>4.1.1</td>
+        <td>-</td>
+    </tr>
+    <tr>
+        <td>3.7</td>
+        <td>0.18.14</td>
+        <td>6.7.0</td>
+        <td>1.8.0</td>
+    </tr>
+    <tr>
+        <td>3.8</td>
+        <td>Active</td>
+        <td>7.1.0</td>
+        <td>1.8.0</td>
+    </tr>
+    <tr>
+        <td>>= 3.9</td>
+        <td>Active</td>
```

Comment:
I was a little confused by Active. Should we say Current ?

Reply:
This is a support matrix which suggests we have active support, so I named Active. But I am also not sure if we should remove the compatibility matrix we had earlier. Any suggestions on this ?

Reply:
I can rename to Current as well, I do not have a strong opinion on keeping it as Active

Reply:
yes, because they are all versions in the matrix, when it says 'Active' it doesn't hit me immediately what is 'Active' versions but I know what 'Current' version is.

Reply:
but otherwise I am fine :) thanks

Reply:
I renamed `Active` to `Latest`. thanks 👍